### ISSUE TITLE
[Bug] No contentDescriptions for Emojis, and no way to close the panel with TalkBack

### ISSUE THREAD

**Issue by MarcoZehe** [tuskyapp/Tusky#937]
When using the TalkBack screen reader, everything in the app is beautifully accessible, has proper labels/contentDescription, except the Emoji panel. Those buttons all are "unlabeled". And the worst part, once you're in, you can't get out. The control to close the panel is either among the unlabeled buttons, or a custom component that TalkBack doesn't even see at all. I was unable to close the Emoji panel when composing a toot. I had to hit Back at the bottom of the screen, save my toot as draft, then reopen the draft and finish composing without the Emoji panel.

* * * *
- Tusky Version: 4.0
- Android Version: Android 8.1.0
- Android Device: Nokia 8 Sirocco
- Mastodon instance (if applicable): n/a

- [x] I searched or browsed the repo’s other issues to ensure this is not a duplicate.

**Comment by connyduck** [comment]
Ok, adding accessibility labels is easy, will fix soon.
But how should we handle the Emoji sheet? It gets closed by clicking on the emoji picker again.
Is it enough to change the label on that button? Or should we close all sheets on back click before closing the whole activity?

**Comment by MarcoZehe** [comment]
Hm, I am not sure. To TalkBack, it looks like the Emoji picker is a full-screen overlay over the whole composer, meaning I no longer feel the text field and the associated buttons for the toot. The top half of my screen feels blank. I am blind, so cannot tell whether that's actually the case, or it is just TalkBack no longer seeing the rest of the UI. Normally, this is correct behavior, as overlays usually are exclusively interactive. However, if you're supposed to still be able to interact with the buttons and the text field of the toot, the overlay is definitely hiding too much here. If that's the case, TalkBack simply doesn't see the button to close the Emoji picker any more. The original button is properly labeled, so if the label doesn't change, but is still present, that's not the issue. But I don't see a labeled button at all any more with TalkBack as soon as the sheet opens.

I think the easiest would simply be to have the Back button at the bottom of the screen simply dismiss the Emoji sheet and return to the keyboard input with all other elements present. Debugging or finding out in the API docs why the overlay hides everything, and how to change that, might be more difficult.
